src/components/panel/panel.js

```diff
@@ -784,6 +785,12 @@ function MdPanelRef(config, $injector) {
    */
   this.isAttached = false;
 
+  /**
+   * Whether the panel is destroyed. This is synchronous. When destroy is called,
+   * isDestroyed is set to true.
+   * @type {boolean}
+   */
+  this.isDestroyed = false;
```

Comment:
This is public, so it needs to be a part of the public documentation, like `isAttached`.